lib/frozen_record/test_helper.rb

```diff
@@ -0,0 +1,35 @@
+module FrozenRecord
+  module TestHelper
+    def self.load_fixture(model_class, alternate_base_path)
+      @cache ||= {}
+
+      ensure_model_class_is_frozenrecord(model_class)
+
+      @cache[model_class] = {
+        old_base_path: model_class.base_path,
+        old_auto_reloading: model_class.auto_reloading,
+      }
+
+      model_class.auto_reloading = true
+      model_class.base_path = alternate_base_path
+    end
+
+    def self.unload_fixtures
+      @cache.each do |model_class, cached_values|
+        ensure_model_class_is_frozenrecord(model_class)
+
+        model_class.base_path = cached_values[:old_base_path]
+        model_class.load_records
+        model_class.auto_reloading = cached_values[:old_auto_reloading]
+      end
+    end
+
+    private
+
+    def self.ensure_model_class_is_frozenrecord(model_class)
+      return if model_class < FrozenRecord::Base
```

Comment:
Ah, I didn't know we could do this. What are you thoughts on using:
``` ruby
return if model_class.ancestors.include?(FrozenRecord::Base)
```

Reply:
It's much less efficient.

Reply:
Which one is much less efficient?

Reply:
`.ancestors.include?` because it allocates and array with all the ancestor chain (parent classes and included / prepended modules), and then sequentially compare them.

`<` walk the ancestor chain without allocating an array directly in C-land: https://github.com/ruby/ruby/blob/7fa16fd9624ca38f2d82c121a9af5f22c87e2697/object.c#L1862

Reply:
🚲 🏠 `.is_a?`

Reply:
It looks like `is_a?` calls https://github.com/ruby/ruby/blob/7fa16fd9624ca38f2d82c121a9af5f22c87e2697/object.c#L861

Both end up calling `class_search_ancestor`, and it _seems_ like the surrounding differences wouldn't matter for performance. So they seem roughly equivalent?

Reply:
Ah :P there's another difference. `is_a?` is meant to be used to check if the class that _an instance_ is of inherits from something.

So `<` it is.